packages/fxa-content-server/app/styles/_layout.scss

```diff
@@ -91,7 +91,7 @@
     cursor: pointer; // Use the correct cursor for anchors without an href
     text-decoration: none;
 
-    @include anchor-default-pseudo-classes();
+    // @include anchor-default-pseudo-classes();
```

Comment:
I think we need this though maybe you meant to uncomment it before pushing. There's a comment further up that mentions this should only affect pages that aren't using TW / our TW component classes yet (they've got an ID of `main-content`). If you check out `/reset_password` for example, the "Remember password? Sign in" link doesn't have hover/active styles.

Reply:
Yes, this was just me experimenting. I am glad you spotted this! Thank you.